### DISCUSSION TITLE
Doubt on how to implement QUEST in a concrete case

### DISCUSSION THREAD

**Discussion by franfrutos** [kurokida/jsQuestPlus#6] [General]
Hello! 

I am interested in using QUEST to perform a staircase procedure prior to a visual attention task to control that different colors that will be used as distractors have a similar luminance, with respect to a base color. The idea would be to define the color of the stimuli using CIElab coordinates, and gradually vary the luminance against that base color. 

I've been reading the paper for this implementation of the method, and the demonstrations in that paper and in this github repository. Since I don't know exactly what distribution to expect for a staircase procedure for luminance changes, I wanted to ask if using the function you use in the paper (weibull) would be a good candidate.

Thank you very much and congratulations for the great work!
Fran

**Comment by kurokida** [comment]
Hi, Fran,
Thank you for your question.

Sorry but I am not familiar with which function is most appropriate in each case. I think you would be better to check previous studies or make preliminary observations using the method of constant stimuli, etc.

Not exactly what you are looking for, but Section *"From Illuminance to Luminance"* of [Watson and Ahumada (2016)](https://www.researchgate.net/publication/305492593_The_pyramid_of_visibility) may be helpful.

Best,
Daiichiro

**Comment by tpronk** [comment]
Hi Fran, in case Watson and Ahumada (2016) does not get you on the right track, we could consider reaching out to the [PsychoPy/JS team](https://discourse.psychopy.org/). They have got quite some experience in applying QUEST-like staircases to visual perpception research.
